README.md

```diff
@@ -17,25 +15,33 @@ W3bstream is a key Layer-2 solution within the IoTeX ecosystem, designed to supp
 The chain of tasks and their ZK-proofs are recorded **on the IoTeX blockchain**, making them accessible for dApps. The actual data, uniquely referenced by the on-chain tasks, remains available for full off-chain verification.
 
 This architecture ensures secure, reliable, and scalable data processing, allowing DePIN dApps to act on verified real-world facts to trigger blockchain-based incentives.
+ 
+
+## Getting Started
+
+### For Embedded Developer
+
+[<u>ioID-SDK</u>](https://github.com/iotexproject/ioID-SDK) repo provides SDK for DePIN hardwares to connect W3bstream and IoTeX ecosystem
+
+
+### For Project Builder 
+
+[<u>Build a custom zk prover for W3bstream</u>](./docs/DEVELOPER_GUIDE.md)
 
-## Running nodes
- > ⓘ **Note**: Joining the W3bstream network as a sequencer or prover node is currently unavailable. Stay tuned for updates in future releases. [Follow us on X](https://x.com/iotex_dev).
 
-## Get Started for Builders
+[<u>Deploy the zk prover to W3bstream</u>](./docs/QUICK_START.md)
 
-[Deploy a simple zk prover to W3bstream →](./docs/QUICK_START.md)
+### For Node operator
 
-[Build your custom zk prover for W3bstream →](./docs/DEVELOPER_GUIDE.md)
 
-## Docker images
+> ⓘ **Note**: Joining the W3bstream network as a sequencer or prover node is currently unavailable. Stay tuned for updates in future releases. [Follow us on X](https://x.com/iotex_dev).
 
-[Sequencer Node →](https://github.com/iotexproject/w3bstream/pkgs/container/w3bstream-sequencer)
+### For Module Integrator
 
-[Prover Node →](https://github.com/iotexproject/w3bstream/pkgs/container/w3bstream-prover)
+ - Storage: [<u>./datasource</u>](./datasource/) folder contains [<u>documentation</u>](./datasource/README.md)  and existing implementations W3bstream can suppport. 
 
-## Contract Deployments
+ - ZK Engine: ZK Service needs to support [Protobuf interface](./vm/proto/vm.proto) to communication with ZK Node
```

Comment:
- **ZK Engine Modules**: To add a new type of ZK prover, please ensure the service implements the [Protobuf interface](./vm/proto/vm.proto) to enable communication with the W3bstream ZK Node.